signIn.html

```diff
@@ -18,29 +18,30 @@
             <img class="image-background" src="./images/bg.jpg" alt="background images">
         </div>
 
-        <form class="form-signup form-signup_move" id="form" submit="sendForm()">
+        <form class="form-signup form-signup_login" id="form" submit="sendForm()">
             <h2 class="name-page">Sign In</h2>
             <div class="wrapper-input wrapper-input_signin">
-                <input class="wrapper-input__data wrapper-input__data_email" type="email"
-                    name="email" placeholder="Email">
+                <input class="wrapper-input__data wrapper-input__data_email" type="email" name="email"
+                    placeholder="Email">
             </div>
             <div class="wrapper-input wrapper-input_signin">
-                <input class="wrapper-input__data wrapper-input__data_passw" type="password"
-                    name="password" placeholder="Password">
-                <button class="wrapper-input__show-pass wrapper-input__show-pass_open" type="button">
-                    <img class="wrapper-input__icon-eye" src="./images/icon-eye.svg">
+                <input class="wrapper-input__data wrapper-input__data_password" type="password" name="password"
+                    placeholder="Password">
+                <button 
+                    class="wrapper-input__show-pass wrapper-input__show-pass_open wrapper-input__show-pass_icon-eye"
+                    type="button">
                 </button>
             </div>
             <div class="wrapper-send-form">
                 <button class="wrapper-send-form__button" type="submit">Sign In ></button>
             </div>
         </form>
         <div class="wrapper-link">
-            <span> Don't have an account?</span>            
+            <span> Don't have an account?</span>
             <a href="./signUp.html">Sign up</a>
```

Comment:
Idk your task, but if it is history api, this is not about it

Reply:
history api - I`ll write in the next step.

Reply:
Resolved